src/phoenix/experimental/evals/models/openai.py

```diff
@@ -22,6 +33,69 @@
 }
 LEGACY_COMPLETION_API_MODELS = ("gpt-3.5-turbo-instruct",)
 logger = logging.getLogger(__name__)
+TQDM_BAR_FORMAT = (
+    "Eta:{eta} |{bar}| {percentage:3.1f}% "
+    "({n_fmt}/{total_fmt}) "
+    "[{elapsed}<{remaining}, {rate_fmt}{postfix}]"
+)
+
+# rate limits conservatively pulled from https://learn.microsoft.com/en-us/azure/ai-services/openai/quotas-limits
+AZURE_OPENAI_TOKEN_RATE_LIMITS = {
+    "gpt-35-turbo": 240000,
```

Comment:
Unfortunately, what people provides to model is the "deployment name" which may or may not be the same as the model that it's deployed with, and we have no visibility to what the real model name is. As a result, this dictionary is essentially useless for Azure. 😔